app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+  <h2 class="my-4 text-color headline">Transactions for <%= @category.name %></h2>
+  <p class="text-color general-text">Total Amount: <%= @total_amount %></p>
+<ul class="list-group">
+  <% @budget_transactions.each do |transaction| %>
+    <li class="list-group-item d-flex justify-content-between category-item align-items-center">
+      <div class="d-flex align-items-center">
+        <span class="transaction-name text-color general-text"><%= transaction.name %></span>
+      </div>
+      <span class="text-color general-text">Amount: <%= transaction.amount %></span>
+    </li>
+  <% end %>
+</ul>
+
+
+  <%= link_to "Add", new_category_transaction_path(@category), class: 'btn secondary-color my-4' %>
+  <%= link_to "Back", categories_path, class: 'btn primary-color' %>
+</div>

```

Comment:
- You have done well so far, but you have not styled this page as it is styled in the design guidelines, please remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Design template:
![wq4rq2w4t](https://github.com/najibullahjafari/BudgetPro/assets/73607512/07c1a391-2ed9-4668-83a4-60605ce04272)

Your design:
- Please ensure that the button is placed at the bottom of the page as it is in the design guidelines
![wwefawef](https://github.com/najibullahjafari/BudgetPro/assets/73607512/8446de52-be18-478e-947d-8952428592d1)